### PR TITLE
fix(examples): Update Opacus loss calculation

### PR DESCRIPTION
**Pull request by alekseibalan**
### Description

Hello dear Flower developers,

Thank you for the fantastic framework.
I've been studying federated learning and tried the Opacus example.
For an untrained model, the expected loss is ln(10)=2.3.
However, the loss reported in the first iteration was 73.7,
which is 32 times greater.

After spending some time investigating the cause and comparing
the Opacus example with Quickstart-PyTorch, I found the missing line
in the task.test() method. Let me kindly put it back.

Please consider accepting.
Thank you.

### Related issues/PRs

No related issues/PRs.

### Explanation

Put back the missing line
```python
loss = loss / len(test_loader)
```

### Checklist

- [x] Implement proposed change
- [ ] Write tests
- [ ] Update [documentation](https://flower.ai/docs/writing-documentation.html)
- [ ] Address LLM-reviewer comments, if applicable (e.g., GitHub Copilot)
- [ ] Make CI checks pass
- [ ] Ping maintainers on [Slack](https://flower.ai/join-slack/) (channel `#contributions`)

### Any other comments?

No other comments.

### CODE REVIEW

**Comment by mohammadnaseri** [comment]
@alekseibalan Thanks for tracking this down. This looks right to me.

Since `CrossEntropyLoss` already returns the mean loss for each batch, the current code ends up returning the sum of all batch losses. Dividing by `len(test_loader)` brings it back to the expected scale and matches the other PyTorch examples we have.

**Review by danieljanes — APPROVED** [review]
Thanks for the contribution @alekseibalan !

**Comment by alekseibalan** [comment]
Thank you
